### PR TITLE
ip_columnrow in SET_CELL and other methods

### PR DESCRIPTION
**Pull request by sandraros**
Permits code footprint reduction as described in #524

### CODE REVIEW

File: src/demos/zdemo_excel40.prog.abap [hunk 1]
```diff
@@ -31,10 +31,7 @@ START-OF-SELECTION.
 *    These marked cells will be used for repeatable rows/columns on printpages
 *--------------------------------------------------------------------*
     lo_worksheet->set_area(
-          ip_column_start = 1
-          ip_column_end   = 20
-          ip_row          = 1
-          ip_row_to       = 100
+          ip_range        = 'A1:T100'
```
Comment (AndreaBorgia-Abo) [inline src/demos/zdemo_excel40.prog.abap:34]: modified call to set_area, but not that to change_area_style: [from the explanation](https://github.com/abap2xlsx/abap2xlsx/issues/524#issuecomment-1001204292) I would have expected at least one of the change_area_style calls to be rewritten to take advantage of ip_range.

**Review by AndreaBorgia-Abo — APPROVED** [review]
LGTM: unit tests are ok, plus the new checker returns the same results as before the PR.

File: src/demos/zdemo_excel40.prog.abap (hunk 1 above)
Comment (sandraros) [reply src/demos/zdemo_excel40.prog.abap:34]: Let's do that. In both `change_area_style`, to push people to use `ip_range`?

Comment (sandraros) [reply src/demos/zdemo_excel40.prog.abap:34]: Done.

**Review by AndreaBorgia-Abo — APPROVED** [review]

**Pull request merged**